### ISSUE TITLE
Minimum throttle  doesn't work at  pwm

### ISSUE THREAD

**Issue by tiriad** [iNavFlight/inav#4201]
I was curious to try the dshot150 protocol so I have loaded the version 2.1 but I tried some protocol **pwm as multishot** and when calibrating the esc does **not respect the minimum values**, if for example the minimum value of throttle is 1150 inav sends throttle from 1000 and the motors already start to spin, and another thing I noticed is that the **osd blinks** and I do not think it is by a state of alarm.

**Comment by tiriad** [comment]
when using the dshot protocol I noticed much more noise in the video signal and even failed me the rth failsafe, I do not know if it could be by some interference with the compass, the quad went to one side and another instead of coming straight and was tested and configured well from previous flights with other protocols.

**Comment by shellixyz** [comment]
Please post a configuration dump. Blinking OSD could mean that the OSD chip on your FC is not compatible with the new OSD code. Please try to read the OSD chip part number.

**Comment by tiriad** [comment]
The osd blinking i think it has been fixed here #4208  and also  I've set the video type fixed to ntsc and no problem. My controller is a f4v3 omnibus and is quite supported so I don't think it's an osd chip incompatibility.
On the other hand in pwm protocols as multishot I still find the **bug**, with the motors calibrated and the throttle minimum set to 1150 the motors start to spin from 1000 **skipping** the minimum throttle from which they start to spin, in dshot this does not happen. All this in the configurator.
